test/nightwatch-tester/integration/data/aktivitet-status.js

```diff
@@ -1,15 +1,15 @@
 export const AktivitetStatus = {
     FORSLAG: 1,
     PLANLEGGER: 2,
-    GJENNOMFORER: 3,
+    GJENNOMFORES: 3,
     FULLFORT: 4,
     AVBRUTT: 5,
 
     properties: {
-        1: { value: 'Forslag' },
-        2: { value: 'Planlegger' },
-        3: { value: 'Gjennomfører' },
-        4: { value: 'Fullført' },
-        5: { value: 'Avbrutt' },
+        1: { value: 'Forslag', selectorId: 'Forslag' },
```

Comment:
Lager vi en hard-kobling mot tekstene som vises i applikasjonen her? Isåfall burde vi kanskje ta en titt på å heller legge inn e2e-hook i koden.

Reply:
value brukes kun for logging og selectorId brukes til å hente riktig xPath. xPathen bruker index for å finne korrekt kolonne hvor aktiviteten skal være. i aktivitet-status.js derimot brukes value også til å validere at det står riktig tekst på merkelappen. Tror jeg må lage en egen sak på å gå igjennom og finne alle stedene det er brukt indexer og tekster og lage hooks på de.